src/vs/workbench/parts/debug/electron-browser/debugService.ts

```diff
@@ -1095,7 +1095,7 @@ export class DebugService implements debug.IDebugService {
 			const breakpointsToSend = this.model.getBreakpoints().filter(bp => this.model.areBreakpointsActivated() && bp.enabled && bp.uri.toString() === modelUri.toString());
 
```

Comment:
Sorry thes not alone or first erthte stabilis
![20141212132321](https://user-images.githubusercontent.com/31952169/30746668-2ef0327c-9fb3-11e7-910a-5c51e4f6ea90.gif)
atione